docker/Dockerfile.gpu

```diff
@@ -1,11 +1,12 @@
-FROM nvidia/cuda:12.2.0-runtime-ubi8 as base
+FROM nvidia/cuda:11.7.1-runtime-ubi8 as base
 
 ARG cnlpt_version
 
 RUN yum -y install python39 python39-pip
-RUN pip3.9 install cython torch
```

Comment:
cython and torch are installed by cnlp-transformers for us (and cnlp-transformers overrides the torch version with its own version requirements anyway). So let's not bother with these.